client/my-sites/email/email-providers-stacked-comparison/provider-cards/google-workspace-card.tsx

```diff
@@ -51,76 +57,58 @@ const getGoogleFeatures = () => {
 };
 
 const googleWorkspaceCardInformation: ProviderCard = {
-	detailsExpanded: true,
-	expandButtonLabel: translate( 'Expand' ),
+	detailsExpanded: false,
+	expandButtonLabel: translate( 'Select' ),
 	onExpandedChange: noop,
 	providerKey: 'google',
 	showExpandButton: true,
-	description: translate(
-		'Professional email integrated with Google Meet and other productivity tools from Google.'
-	),
+	description: translate( 'Productivity tools and mailbox from Google.' ),
 	logo: { path: googleWorkspaceIcon, className: 'google-workspace-icon' },
 	productName: getGoogleMailServiceFamily(),
 	features: getGoogleFeatures(),
 };
 
 const GoogleWorkspaceCard: FunctionComponent< EmailProvidersStackedCardProps > = ( props ) => {
-	const { currencyCode = '', domain, gSuiteProduct, selectedDomainName } = props;
+	const {
+		currencyCode = '',
+		detailsExpanded,
+		domain,
+		gSuiteProductMonthly,
+		gSuiteProductYearly,
+		onExpandedChange,
+		selectedDomainName,
+		termLength,
+	} = props;
 	const googleWorkspace: ProviderCard = googleWorkspaceCardInformation;
+	googleWorkspace.detailsExpanded = detailsExpanded;
```

Comment:
I wonder if this mutation is intentional? The previous line makes an alias of `googleWorkspaceCardInformation` called `googleWorkspace`, and then this line changes the `detailsExpanded` property of that object, _also changing the same property_ of the global `googleWorkspaceCardInformation` object (because they point to the same object).

If you want `googleWorkspace`  to be a _copy_ of the global object instead, you could change the previous line to something like `const googleWorkspace = {...googleWorkspaceCardInformation};`